source/options.html

```diff
@@ -5,22 +5,24 @@
 <link rel="stylesheet" href="options.css">
 <form id="options-form" class="detail-view-container">
 	<p>
-		<strong>Information</strong><br>
-		Visit the <a href="https://github.com/refined-github/refined-github/issues/3543">welcome page</a> to learn about updates, debugging, and GitHub Enterprise. Follow or sponsor <a href="https://github.com/sponsors/fregante">@fregante</a> if Refined GitHub helps you work more efficiently. 🍻
+		<strong>👋 Information</strong><br>
+		Visit the <a href="https://github.com/refined-github/refined-github/issues/3543">welcome page</a> to learn about updates, debugging, and GitHub Enterprise.
+		You can <a href="https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf/reviews" id="rate-link">rate Refined GitHub</a> to help others find it.
```

Comment:
As requested in https://github.com/refined-github/refined-github/issues/4008#issuecomment-823849930

Updated via JS on non-Chrome

Reply:
Like!